src/main/java/uk/gov/ons/census/casesvc/service/CaseProcessor.java

```diff
@@ -49,7 +50,8 @@ public void emitCaseCreatedEvent(Case caze) {
     responseManagementEvent.setEvent(event);
     responseManagementEvent.setPayload(payload);
 
-    rabbitTemplate.convertAndSend(emitCaseEventExchange, "", responseManagementEvent);
+    rabbitTemplate.convertAndSend(
+        outboundExchange, CASE_UPDATE_ROUTING_KEY, responseManagementEvent);
```

Comment:
same as above
```suggestion
        outboundExchange, CASE_CREATE_ROUTING_KEY, responseManagementEvent);
```

Reply:
this is what's in the spec here: https://collaborate2.ons.gov.uk/confluence/display/CATD/RM+Event+Model

Has been agreed with Rob Smart, Phil Whiles & Ella Cook... so gonna mark this as resolved.